scripts/spack/configs/darwin/compilers.yaml

```diff
@@ -2,51 +2,14 @@ compilers:
 - compiler:
     environment: {}
     extra_rpaths: []
-    flags: {}
+    flags:
+      cflags: -Wno-implicit-function-declaration
     modules: []
-    operating_system: elcapitan
+    operating_system: catalina
```

Comment:
We could get around this by copying and pasting this several times. I created a Spack issue but there are probably technical reasons to not allow this to be base value or undefined...

https://github.com/spack/spack/issues/23334

Reply:
Uberenv will also try `spack compiler find` if `compilers.yaml` is empty - though I'm not sure if this would find the Fortran compilers, and I'm definitely sure that it wouldn't set that `-Wno` flag you ended up needing

Reply:
I should probably try to move that flag into the hdf5 package itself.